activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -214,6 +214,7 @@ def initialize(types, name, temporary, options, as = nil)
         @name = name
       end
 
+      # Returns an array of ColumnDefinitions for the columns of the table.
```

Comment:
@carlosantoniodasilva fair enough. I'm currently reworking how we reference classes and methods in our documentation. I'll apply this patch for now and see how to improve the situation in my branch.